src/nextjournal/clerk.clj

```diff
@@ -50,7 +50,8 @@
                     :else
                     file-or-ns)
              doc (try (merge opts
-                             {:nav-path (webserver/->nav-path file-or-ns)}
+                             {:file file
```

Comment:
Can we normalize `file` as string here? It seems to make troubles in the header: 

<img width="600" alt="CleanShot 2023-10-12 at 16 33 56@2x" src="https://github.com/nextjournal/clerk/assets/1078464/782eebda-867b-4f58-a247-e9e824e13c9d">

when calling `show!` against a namespace or a resource e.g.
```clojure
(clerk/show! (io/resource "nextjournal/clerk/tap.clj"))

(clerk/show! 'nextjournal.clerk.tap)
```

s `:file` consumed anywhere else?

Reply:
🤔 ach nö, the doc returned from parse will always assoc a `:file` key with the same value. Maybe we should just handle it in the view? And there should no need to assoc it here?